src/components/Sourcing/PaStageManufacturer.jsx

```
@@ -70,116 +73,249 @@ export default function PaStageManufacturer(props) {
+                                ? "negative"
+                                : ""
+                        }`}
+                      >
+                        {pa.total_variance === 0
+                          ? pa.total_variance?.toLocaleString()
+                          : pa.total_variance?.toLocaleString()}
+                      </div>
+                    </div>
+                  </div>
+                ) : (
+                  <div className="supplier-summaryManufacturer-row p-2">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)